lessons/codap-transformers2/langs/en-us/pages/notes-sort-transformer.adoc

```diff
@@ -0,0 +1,14 @@
+= Using Transformers: Sort, Filter, Transform Attribute, Build Attribute
```

Comment:
Shouldn't this be in codap-transformers1?

Reply:
yeeah, things got shuffled and need a bit of reorganization.